pkg/restore/systable_restore.go

```diff
@@ -61,15 +72,15 @@ func (rc *Client) RestoreSystemSchemas(ctx context.Context, f filter.Filter) {
 	tablesRestored := make([]string, 0, len(originDatabase.Tables))
 	for _, table := range originDatabase.Tables {
 		tableName := table.Info.Name
-		if f.MatchTable(sysDB, tableName.O) {
+		if f.MatchTable(temporaryDB.O, tableName.O) {
```

Comment:
Seems after this, users must use `-f __TiDB_BR_Temporary_mysql.xxx` to specify which tables to be restored. 🤔

Reply:
yes... but for now we will filter `!__TiDB_BR_Temporary_mysql.xxx` in[ L45](__TiDB_BR_Temporary_mysql.xxx)...

Reply:
Once the user specifies any table filter, seems that rule would be overridden. So it would be OK? 🤔

Reply:
I didn't get it. 

In L45, we filtered out other DBs only keep the `temporaryDB` which means we can not meet `sysDB` here.

Reply:
which means when we do 
`br backup db --db mysql xxx`
and then `br restore db --db mysql` we will get nothing to restore...

Reply:
Well, I think we'd use original database name everywhere. L45 is buggy😢, the current version is fine.